### PR TITLE
Add Pulsar Community Weekly Update - 2021-06-21~ 2021-06-27

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2021/2021-06/2021-06-28-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+- [Configuration] Add missing configuration entries.
+
+    https://github.com/apache/pulsar/pull/11095 ([@junquero](https://github.com/junquero))
+
+### Notable Bug Fix
+
+- [Client] Fix the `NoClassDefFoundError` that the `io.airlift.compress.lz4.UnsafeUtil` class cannot be initialized.
+
+    https://github.com/apache/pulsar/pull/10983 ([[@newur](https://github.com/newur))
```
Comment (BewareMyPower) [inline content/2021/2021-06/2021-06-28-pulsar-weekly.md:107]:
```suggestion
    https://github.com/apache/pulsar/pull/10983 ([@newur](https://github.com/newur))
```

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

**Review by BewareMyPower — APPROVED** [review]

File: content/2021/2021-06/2021-06-28-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+### Blog / Article
+
+- Pulsar Hackathon 2021 Winners Announced
+
+    - https://streamnative.io/en/blog/community/2021-06-22-pulsar-hackathon-2021-winners-announced
+
+- Highlights from The Pulsar Virtual Summit North America 2021
+
+    - https://streamnative.io/en/blog/community/2021-06-22-highlights-from-the-pulsar-virtual-summit-north-america-2021
```
Comment (Jennifer88huang) [inline content/2021/2021-06/2021-06-28-pulsar-weekly.md:241]: It's good to include StreamNative blogs, if there are more good quality blogs from the Pulsar community, please also include them to show diversity.

File: content/2021/2021-06/2021-06-28-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+    https://github.com/apache/pulsar/pull/11052 ([@gaoran10](https://github.com/gaoran10))
+
+- [Topic Policy] Fix the `SystemTopicBasedTopicPoliciesService` race condition.
+
+    https://github.com/apache/pulsar/pull/11097 ([@codelipenghui](https://github.com/codelipenghui))
+
+### Event / News
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
```
Comment (Jennifer88huang) [inline content/2021/2021-06/2021-06-28-pulsar-weekly.md:223]: There will be big changes to TGIP name and episodes soon, you can add it below other impactful events in the future.
For example, you can introduce Pulsar 101 first and then TGIP.

File: content/2021/2021-06/2021-06-28-pulsar-weekly.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+- [Topic Policy] Fix the issue that the topic-level retention policy does not work.
+
+    https://github.com/apache/pulsar/pull/11021 ([@hangc0276](https://github.com/hangc0276))
+
+- [Functions] Fix the build failure caused by the SpotBugs.
+
+    https://github.com/apache/pulsar/pull/10792 ([@Jason918](https://github.com/Jason918))
+
+- [Build] Fix the compilation error caused by the the number of incorrect parameters.
```
Comment (Anonymitaet) [inline content/2021/2021-06/2021-06-28-pulsar-weekly.md:209]:
```suggestion
- [Build] Fix the compilation error caused by the number of incorrect parameters.
```

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**